packages/api/models/nomination.js

```diff
@@ -1,4 +1,5 @@
 'use strict';
+const { sendVerification } = require('../helper/mailer.js');
```

Comment:
Looks like this is imported but not used. Is needed to just set up the nodemailer or will be used in the future?

Reply:
Will be used in the future. When a nomination is created the mailer function will be used on a hook. Will take out the import for now 👍